apps/main/src/llamalend/features/manage-soft-liquidation/hooks/useImproveHealthTab.ts

```diff
@@ -20,6 +20,7 @@ export function useImproveHealthTab(params: MarketParams): ImproveHealthProps {
         stateCollateral: '0' as Decimal,
         userCollateral: '0' as Decimal,
         userBorrowed: debt,
+        isFull: false,
```

Comment:
Could we add a todo comment here to so we don't forget to add the missing isFull check? Technically users can make a full repay here.